app/frontend/schools/courses/components/CurriculumEditor.res

```diff
@@ -235,23 +235,23 @@ let make = (
             <div className="inline-block relative w-auto md:w-64">
               <select
                 onChange={event => {
-                  let level_name = ReactEvent.Form.target(event)["value"]
-                  send(SelectLevel(Level.selectLevel(state.levels, level_name)))
+                  let level_id = ReactEvent.Form.target(event)["value"]
+                  send(SelectLevel(Level.selectLevel(state.levels, level_id)))
                 }}
-                value={currentLevel |> Level.name}
+                value={currentLevel->Level.id}
```

Comment:
```suggestion
                value={Level.id(currentLevel)}
```
Do make this change across the PR; 
https://github.com/pupilfirst/pupilfirst/pull/1319#discussion_r1226828518